docs-ref-services/latest/resourcemanager-mediaservices-readme.md

```diff
@@ -1,14 +1,13 @@
 ---
 title: Azure Resource Manager MediaServices client library for Java
 keywords: Azure, java, SDK, API, azure-resourcemanager-mediaservices, mediaservices
+description: This package contains Microsoft Azure SDK for MediaServices Management SDK. This Swagger was generated by the API Framework.
 author: joshfree
```

Comment:
all of the author and ms.author fields should be seeded from the .github/CODEOWNERS file in the azure-sdk-for-java repo.